pycaret/internal/tabular.py

```diff
@@ -10333,6 +10333,28 @@ def convert_model(estimator, language: str = "python") -> str:
         )
 
 
+def eda(data=None, target=None, display_format = 'bokeh', **kwargs):
+
+    """
+    Function to generate EDA using AutoVIZ library.
+    """
+    
+    if data is None:
+        try:
+            data = get_config('data_before_preprocess')
+        except:
+            raise ValueError('When running EDA function before setup, you must pass the pandas.DataFrame explicitly.')
+
+    if target is None:
+        try:
+            target = get_config('prep_pipe')[0].target
+        except:
+            raise ValueError('When running EDA function before setup, you must pass the target column name explicitly.')
+
+    from autoviz.AutoViz_Class import AutoViz_Class
```

Comment:
Can we have a try except import at the top of the function, like in other integrations?